types/pico8parse/lib/ast.d.ts

```diff
@@ -0,0 +1,337 @@
+export interface Base<TType extends string> {
```

Comment:
This file implies that you can import from `"pico8parse/lib/ast"`, which does not actually exist on npm, so unfortunately everything will have to go in the index file.